dbft.go

```diff
@@ -489,6 +489,19 @@ func (d *DBFT) onChangeView(msg payload.ConsensusPayload) {
 }
 
 func (d *DBFT) onCommit(msg payload.ConsensusPayload) {
+	existing := d.CommitPayloads[msg.ValidatorIndex()]
+	if existing != nil {
+		if !existing.Hash().Equals(msg.Hash()) {
+			d.Logger.Warn("rejecting commit due to existing",
```

Comment:
The node committed at view 0 can't commit at view 1. Well, I know it technically is correct for dbft 2.0.

Reply:
Yes, we'll need to review the whole dbft code *very carefully* on the way to 2.1. There's a set of snippets that are valid for 2.0 but invalid for 2.1. I'd suggest to keep this change for now. Or not?

Reply:
We keep it, it makes sense for 2.0.